astropy/time/tests/test_basic.py

```diff
@@ -1346,7 +1346,15 @@ def test_TimeFormat_scale():
 
 
 @pytest.mark.remote_data
-def test_scale_conversion():
+def test_scale_conversion(monkeypatch):
+    monkeypatch.setattr('astropy.utils.iers.conf.auto_download', True)
+    with iers.earth_orientation_table.set(iers.IERS_B.open()):
```

Comment:
Note: I was very surprised about the single line in the test as written so rewrite to make it similar to the original purpose of this test (which was removed because it seemed tricky to do; indeed, resulting in a discussion that was not actually resolved... https://github.com/astropy/astropy/pull/6606/files#r140902373).

Reply:
Hmm, that new test for exception with bundled table doesn't need to be marked as remote data, right? Can it be in its own function? Or does it still somehow needs internet?

Reply:
Looking at this again, there really is no point in testing the exception, so I removed it again. Instead, I now added a comment so it is clearer why the test exists.